intellst-api/src/Services/IdentifiedCaseHandler.php

```diff
@@ -81,4 +102,92 @@ public function updateIdentifiedCaseAllowEntrance(AllowEntranceDTO $dto, Identif
 
         return $errors;
     }
+
+    public function getIdentifiedCase(): array
+    {
+        $user = $this->userHandler->getList();
+        $temperature = $this->returnUserEnterprise($user)->temperature;
+        $data = $this->returnUserEnterprise($user)->restrictionPeriod;
+        $users = $this->identifiedCaseRepository->getNewIdentifiedCase($data, $temperature);
+        $arr = [];
+        foreach ($users as $identifiedCase) {
+            $identifiedCaseDTO = $this->identifiedCaseTransformer->transformEntityToDTO($identifiedCase);
+            $arr[] = $identifiedCaseDTO;
+        }
+
+        return $arr;
+    }
+
+    public function getReturnAttempts(IdentifiedCaseDTO $dto): bool
+    {
+        $temperature = $this->getEnterprise($dto)->temperature;
+        $day = $this->getEnterprise($dto)->restrictionPeriod;
+        $uuid = $dto->uuid;
+        $list = $this->identifiedCaseRepository->getReturnAttempts($day, $temperature, $uuid);
+        $arr = [];
+        foreach ($list as $identifiedCase) {
+            $identifiedCaseDTO = $this->identifiedCaseTransformer->transformEntityToDTO($identifiedCase);
+            $arr[] = $identifiedCaseDTO;
+        }
+        if (!empty($arr)) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    public function entranceAllowed(IdentifiedCaseDTO $dto): bool
+    {
+        $uuid = $dto->uuid;
+        $list = $this->identifiedCaseRepository->entranceAllowed($uuid);
+        $arr = [];
+        foreach ($list as $identifiedCase) {
+            $identifiedCaseDTO = $this->identifiedCaseTransformer->transformEntityToDTO($identifiedCase);
+            $arr[] = $identifiedCaseDTO;
+        }
+        if (!empty($arr)) {
+            return true;
+        } else {
+            return false;
+        }
+    }
+
+    private function getEnterprise(IdentifiedCaseDTO $dto): EnterpriseDTO
+    {
+        $enterprise = $this->enterpriseRepository->find($dto->enterprise);
+        $enterpriseDTO = $this->enterpriseTransformer->transformEntityToDTO($enterprise);
+
+        return $enterpriseDTO;
+    }
+
+    private function returnUserEnterprise(UserDTO $dto): EnterpriseDTO
```

Comment:
`getUserEnterprise`
Also, why have 2 methods? Isn't it simplied to have 1 method that return EnterpriseDTO by enterprise ID? You can have just `getEnterprise(int $id)`